vdb/__init__.py

```diff
@@ -1861,6 +1862,8 @@ def do_bp(self, line):
 
         self.vprint(" [ Breakpoints ]")
         for bp in self.trace.getBreakpoints():
+            if bp.untouchable:  # don't list untouchable bp's
+                continue
```

Comment:
adding the ability to make bp's "untouchable" (because we need to protect this LD hook for library loading)

Reply:
we could do a `bp -A` to list all. 
untouchable is mostly so that `bp -C` doesn't get rid of bp's that VDB sets like this.

Reply:
done.